xlators/mgmt/glusterd/src/glusterd-snapshot.c

```diff
@@ -8471,6 +8471,13 @@ glusterd_snapshot_restore_postop(dict_t *dict, int32_t op_ret, char **op_errstr,
         /* After restore fails, we have to remove mount point for
          * deactivated snaps which was created at start of restore op.
          */
+        if (!volinfo) {
+            gf_msg(this->name, GF_LOG_ERROR, EINVAL, GD_MSG_VOLINFO_GET_FAIL,
```

Comment:
Prefer to use `gf_smsg` when adding new log statements

Reply:
Thank you for pointing it out. I used `gf_msg` as it is used extensively throughout this file. I also checked the implementations of both and didn't find that much of a difference as `gf_smsg` ultimately calls `gf_msg` by setting `ERRNUM` as 0.

Can you help me understand the difference between the two? Also, Should I then replace all the occurrences of `gf_msg` in this file with `gf_smsg`

Reply:
`gf_smsg` is for structured logging when you have dynamic fields along with static message in logs.

Reply:
Thank you! So I guess `gf_msg` is sufficient here, right?